gatsby-browser.js

```diff
@@ -0,0 +1,6 @@
+try {
+  // eslint-disable-next-line import/no-internal-modules
+  require("@jobber/fonts/dist/index.css");
```

Comment:
Depending on how fast we can put up the jobber fonts package update, this could stay here or get patched up after the fact. This is mostly an internal jobber thing anyway so no biggie on that account